### PR TITLE
Eth2 rebrand

### LINKED ISSUE #5084: Trilemma text labels are not visible
**Describe the bug**

The text labels for 'Security', 'Decentralisation', and 'Scaling' are not visible on the [Trilemma](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Trilemma.js). However, they are showing up in the DOM.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://ethereum.org/en/eth2/vision/'
3. Scroll down to 'The challenge of decentralized scaling'

**Expected behavior**

The labels should be visible next to the circular buttons.

**Screenshots**

Screenshot 2022-01-13 at 11 55 01

Screenshot 2022-01-13 at 11 38 10

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Revamp of our site content to replace usage of "Eth2", given that once the merge takes place, there will no longer be two distinct Ethereum networks or concepts of "Eth1" and "Eth2" - there will just be the Ethereum network we all know in love.

Preview deploy:
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/merge/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/beacon-chain/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/shard-chains/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/vision/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/get-involved/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/staking/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/staking/deposit-contract/
https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/developers/docs/nodes-and-clients/

## Related Issue

#4215
[Fixes #5084]

## Remaining todos

_Inexhaustive list, please feel free to comment on what other updates are needed!_

- [x] Rework [Eth2 homepage](https://ethereum.org/en/eth2/) content, including a new section that explains "What happened to Eth2?"
- [x] Finals edits to "What happened to Eth2" copy
- [x] Rework [Eth2 beacon chain page](https://ethereum.org/en/eth2/beacon-chain/) content
- [x] Rework [Eth2 merge page](https://ethereum.org/en/eth2/merge/) content
- [x] Rework [Eth2 shard chains page](https://ethereum.org/en/eth2/shard-chains/) content
- [x] Rework [Eth2 vision page](https://ethereum.org/en/eth2/vision/) content
- [x] Rework [Eth2 staking page](https://ethereum.org/en/eth2/staking/) content
- [x] Rework [Eth2 get involved page](https://ethereum.org/en/eth2/get-involved/) content
- [x] Rework [Eth2 bug bounty page](https://ethereum.org/en/eth2/get-involved/bug-bounty/) content
- [x] Rework [Eth2 deposit contract page](https://ethereum.org/en/eth2/deposit-contract/) content
- [x] Rename `*-eth2-*.json` files (for intl support)
- [x] Rename Eth2 components
- [x] Rename Eth2 images
- [x] Update [nodes and client page](https://ethereum.org/en/developers/docs/nodes-and-clients/) documentation to delineate between execution clients & consensus clients #5064 
- [x] Add 301 redirect rules (e.g. from /eth2/* to /upgrades/*)
- [x] Finalize intl strings

## PR contains
- Reorganizes file structure (build paths for webpages) to be upgrades or staking instead of eth2
- `eth2` template changed to `upgrade` template
- Update any links on website from `/eth2/` to `/upgrades/` or `/staking/`
- Update `/upgrades/vision/`, `/upgrades/get-involved/`, `/upgrades/`, `/upgrades/get-involved/bug-bounty/` copy away from "eth2"
- Update `/developers/docs/nodes-and-clients/` copy away from "eth2"
- [Updates intl json string keys and filenames](https://github.com/ethereum/ethereum-org-website/pull/5063) changes all eth2 json files to be either staking or upgrades, as well as their translation keys. Updates the components to also use these new translation strings.
- [Update glossary for eth2 rebrand](https://github.com/ethereum/ethereum-org-website/pull/5061) updates glossary definitions away from eth2, and where applicable refers to either Ethereum upgrades or consensus layer.
- [Add callouts and/or banner on "What happened to Eth2"](https://github.com/ethereum/ethereum-org-website/pull/5057) updates the upgrade banner to "What happened to Eth2"
- [Rename images](https://github.com/ethereum/ethereum-org-website/pull/5056) changes file names for eth2 image assets to upgrade_{assetname}. Update where these files were used to use new names.
- [Add content on “Eth2 scams”](https://github.com/ethereum/ethereum-org-website/pull/5060) Adds "Eth2 scams" to security and scams page
- [Update eth2 redirects](https://github.com/ethereum/ethereum-org-website/pull/5059) adds redirect support away from /eth2/ to /upgrades/
- [Update pages using upgrade template](https://github.com/ethereum/ethereum-org-website/pull/5058) Remove eth2 references in /upgrades/beacon-chain/, /upgrades/merge/, and /upgrades/shard-chain/
- [Rename Eth2 components](https://github.com/ethereum/ethereum-org-website/pull/5062) change Eth2 components implementation/name away from Eth2 and use either Upgrade, or completely move the Eth reference in the name. Update any files using these components. Updated some articles and add use for UpgradeArticles.
- [Updates nodes-and-clients copy](https://github.com/ethereum/ethereum-org-website/pull/5064) Updates terms on nodes-and-clients page to use "execution" and "consensus" clients and drafted table for consensus layer clients
- [Eth2 rebrand: Nav](https://github.com/ethereum/ethereum-org-website/pull/5091) Update nav from `Eth 2.0` to `Ethereum upgrades`

### CODE REVIEW

**Comment by samajammin** [comment]
@corwintines @wackerow great work!

Noticing it still stays "Ethereum 2.0" in the site nav:
Image 2022-01-13 at 8 25 40 AM

Please make sure to run a final grep for "Eth2"/"Ethereum 2" & make sure all references we want to still be there (e.g. places where we explain the evolution).

File: src/content/developers/tutorials/run-node-raspberry-pi/index.md
Comment (samajammin) [inline src/content/developers/tutorials/run-node-raspberry-pi/index.md:20]:
```suggestion
Besides, [consensus clients](/upgrades/get-involved/#eth2-clients) don’t support 32 bits binaries so using Raspbian would exclude the Raspberry Pi 4 from running an Eth 2.0 node (and the possibility of staking).
```

Let's make sure we check for all mentions of "Eth2"/"Eth 2.0"/"Ethereum 2" etc.

Comment (samajammin) [reply src/content/developers/tutorials/run-node-raspberry-pi/index.md:20]: Anchor links (e.g. #eth2-clients here) should also be updated.

File: src/content/security/index.md
Comment (samajammin) [inline src/content/security/index.md:259]:
```suggestion
With [the merge](/upgrades/merge/) coming in 2022, scammers have taken advantage of the confusion around the term "Eth2" to try and get users to redeem their ETH for an "ETH2" token. There is no "ETH2" or any other new token introduced with the merge. The ETH that you own today will continue to be the same ETH after the merge, and there is no need to do any swaps of your ETH for the merge.
```

Comment (samajammin) [reply src/content/security/index.md:259]: We should add a link to the /upgrades/ homepage section that explains what happened to Eth2.

**Comment by samajammin** [comment]
@corwintines @wackerow another thought related to the glossary updates - what do you folks think about adding:
- Client
- Execution client
- Consensus client

Might be helpful to explain & distinguish these terms. Gives us another opportunity to clarify the evolving terminology.

**Comment by minimalsm** [comment]
The read more sections on the beacon chain and sharding page are displaying correctly. I think their translation strings were accidently deleted?

Screenshot 2022-01-14 at 13 23 13

Screenshot 2022-01-14 at 13 23 01

**Comment by samajammin** [comment]
Some translation strings on the client section are broken:

Image 2022-01-16 at 4 27 50 PM

https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/get-involved/#clients

File: src/pages/upgrades/get-involved/index.js
Comment (samajammin) [inline src/pages/upgrades/get-involved/index.js]: This should be deleted - looks like a duplicate array?

Comment (samajammin) [reply src/pages/upgrades/get-involved/index.js]: @corwintines @wackerow please clean up all the console warnings while we're at it:
```
warning /Users/samrichards/code/ef/ethereum-org-website/src/components/Layout.js
  96:6  warning  React Hook useEffect has a missing dependency: 'props.location.hash'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/components/Morpher.js
  152:6  warning  React Hook useEffect has missing dependencies: 'morphTimeout', 'morpher', 'state.text', and 'state.words'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/components/YouTube.js
  24:7  warning  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title

/Users/samrichards/code/ef/ethereum-org-website/src/pages/developers/learning-tools.js
  87:6  warning  React Hook useEffect has a missing dependency: 'sandboxes'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/pages/upgrades/get-involved/index.js
  247:9  warning  'consensusClients' is assigned a value but never used  no-unused-vars
```

Comment (wackerow) [reply src/pages/upgrades/get-involved/index.js]: Already put up a separate PR (#5097) for this cleanup =)

**Review by wackerow — APPROVED** [review]

**Comment by wackerow** [comment]
Note, the _Medalla Data Challenge_ and _Staking Community Grants_ pages were not fully updated as they have been deprecated. They're not linked to, but they do have working URLs. They're significantly outdated at this point and contain broken links, like to the Medalla launchpad. Should we consider removing these files entirely? 

Aside, I did note misuse of a couple components in the Medalla file, so I updated that (fixes a few console warnings)

**Review by corwintines — APPROVED** [review]

**Comment by wackerow** [comment]
To everyone: Going to bring this in, but we're still looking for feedback on these changes.

Please feel free to continue to comment on this with thoughts, and/or open new issues which we can use to make adjustments here moving forward. 🙏🏻 

In the meantime this will unblock `dev` branch and enable us to get merge conflicts fixed for other PRs.

**Comment by timbeiko** [comment]
Awesome work 👏 looking at the pages, here is a bit of feedback:

https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/

> 'Eth1' is now the 'execution layer', which handles transactions and data.

I would say "transactions and execution", because "data", while technically true, can be confusing with sharding coming soon. 

https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/shard-chains/
> Shard chains spread the network's load across 64 new chains.

This is not true of the latest designs. I would recommend rephrasing as "Shard chains provide extra, cheaper, storage layers for applications and rollups to store data"

> They make it easier to run a node by keeping hardware requirements low.
This is not really accurate, I would just remove. If you want something along those lines, I'd say "They enable layer 2 solutions to offer low transaction fees while leveraging the security of Ethereum"

> These shards will give Ethereum more capacity to store and access data, but they won’t be used for executing code. The details of that are still being figured out.

It's unlikely that the shards ever execute code in the current roadmap. I think you can just delete the last sentence here. 

> Shard chains version 1: data availability

This is really the only part of sharding that is still on the roadmap. The second version has been put on hold for now. Perhaps it'd be worth getting a researcher to review this page and submit a PR to make sure it's accurate. cc: @dankrad

https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/upgrades/get-involved/

> Production consensus clients

This list is a bit odd. I think you could have them all as productiuon clients, and remove Cortex, which is deprecated. 
Given that post-merge, an execution client will also be needed, it might be worth listing them on the same page. The ones supporting the merge for now are: Besu, Erigon, Geth, Nethermind


https://ethereumorgwebsitedev01-eth2rebrand.gtsb.io/en/developers/docs/nodes-and-clients/#execution-clients

> EXECUTION CLIENTS (FORMERLY 'ETH1 CLIENTS')

I'm not sure we want to include OE in the table while its status is uncertain. I would perhaps move it to the last row and add a note about it having been announced as deprecated. 

Geth doesn't support fast sync anymore: they have moved to snap sync. Other clients might have support for it too!
I don't think Erigon supports fast sync. 


That's it! Again, great job on this, Awesome to see it live :-D

**Comment by MicahZoltu** [comment]
In the `What happened to 'Eth2?'` block, I would like to see something that gives a bit of context on why the name `Eth2` is bad and confusing and leads to scams.  In my experience over the past few months trying to get people to change their nomenclature, I have learned that a *lot* of people look to ethereum.org for guidance, and it will make convincing third parties (like Coinbase) much easier if we explain to them that this change isn't just some frivolous bike shedding thing but instead is meant to protect users from scams and confusion.

**Comment by dankrad** [comment]
Yes agreed. It's best to remove execution shards from the roadmap as this only creates confusion. Data shards with rollups are better in all dimensions so barely anyone is thinking about this now.

**Pull request merged**
